pkg/testutil/validate_headers_test.go

```diff
@@ -116,14 +116,14 @@ func TestValidate(t *testing.T) {
 		t.Fatal("error running Validate", err)
 	}
 
-	for _, filename := range *non {
-		t.Logf("nonvalid file: %s", filename)
-	}
-
 	if non == nil {
 		t.Fatal("test did not find any bad files")
```

Comment:
This doesn't seem right (no your changes), but why would we fail if there were no bad files?

Reply:
In the test-case, we are creating some good files and some bad files initially as a test data and it is expected from `Validate()` func that it should return `nonconformingFilesPtr`. So if `non==nil` it means our code is not catching bad files and   hence we are failing